tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -54,7 +54,7 @@ Create with a directory as a volume
     Should Contain  ${output}  Error response from daemon: Bad request error from portlayer: vSphere Integrated Containers does not support mounting directories as a data volume.
```

Comment:
Commenting here since I can't seem to add a comment against line 48:

```
docker %{VCH-PARAMS} logs $(docker %{VCH-PARAMS} start $(docker %{VCH-PARAMS} create -v test-named-vol:/testdir busybox /bin/df -Ph)
```

should use `${busybox}`